packages/hardhat/contracts/BatchRegistry.sol

```diff
@@ -30,6 +30,7 @@ contract BatchRegistry is Ownable {
         _;
     }
 
+
```

Comment:
This change is also unrelated to your personal page, please remove it from the PR.